js_modules/dagit/packages/app/public/index.html

```diff
@@ -6,8 +6,8 @@
     <meta name="theme-color" content="#000000" />
 
     <!-- A target element to insert a custom path prefix, replaced server-side. -->
-    <script type="application/json" id="path-prefix">
-      {"pathPrefix": "__PATH_PREFIX__"}
+    <script type="application/json" id="initialization-data">
+      {"initializationData": {"pathPrefix": "__PATH_PREFIX__"}, {"telemetryEnabled": "__TELEMETRY_ENABLED__"}}
```

Comment:
No need to wrap this and key it on `initializationData` -- you can just add the new key to the existing object.